src/main/webapp/i18n/de/programmingExercise.json

```diff
@@ -379,6 +380,37 @@
                         "repository": "Repository",
                         "totalItems": "Insgesamt {{count}} Element(e)",
                         "error": "Beim Laden der betroffenen Studierenden ist ein Fehler aufgetreten."
+                    },
+                    "feedbackDetailChannel": {
+                        "title": "Erstellen Sie einen Feedback-Diskussionskanal",
+                        "studentNumber": "Anzahl der Studierende die dem Kanal hinzugefügt werden: {{count}}",
+                        "description": "Dieser Kanal soll Diskussionen über spezifisches Feedback zur bereitgestellten Übung erleichtern. Es ermöglicht allen betroffenen Studierenden und Lehrenden, zusammenzuarbeiten und das Feedback effektiv anzugehen.",
+                        "label": "Kanalname",
+                        "channelVisibility": "Kanal Sichtbarkeit",
+                        "visibilityPublic": "Öffentlich",
+                        "visibilityPrivate": "Privat",
+                        "announcementChannel": "Ankündigungskanal",
+                        "announcementChannelYes": "Ja",
+                        "announcementChannelNo": "Nein",
+                        "createAndNavigateLabel": "Erstellen und Navigieren:",
+                        "createAndNavigateDescription": "Erstellt den Kanal und navigiert automatisch zum Kanal.",
+                        "createChannelLabel": "Kanal erstellen:",
+                        "createChannelDescription": "Erstellt den Kanal, ohne die Seite zu verlassen.",
+                        "createAndNavigateButton": "Erstellen und Navigieren",
```

Comment:
Can we merge createAndNavigateButton and createAndNavigateLabel? (same for englisch json file)